x-pack/plugins/security_solution/public/common/components/top_n/top_n.tsx

```diff
@@ -38,11 +34,12 @@ const CloseButton = styled(EuiButtonIcon)`
 
 const ViewSelect = styled(EuiSuperSelect)`
   z-index: 999999;
-  width: 155px;
+  width: 170px;
 `;
 
 const TopNContent = styled.div`
   margin-top: 4px;
+  margin-right: 5px;
```

Comment:
nit: Files like this one still contain examples of hard coded values for margins (i.e. the `margin-top: 4px;` on the line above this new line). A pattern later emerged (described below) that uses constants defined by EUI whenever they are applicable.

The actual values for those constants are viewable in `node_modules/@elastic/eui/dist/eui_theme_dark.json`, and may change as EUI evolves.

In general, when adding new styles (or updating existing ones), consider using the following convention (in `styled` components) that uses the EUI constants:

```
margin-right: ${({ theme }) => theme.eui.euiSizeXS};
```

The EUI constants wont be applicable in every case, but when they are, prefer them over hard coded values.